admin/edit-user.html

```diff
@@ -3,10 +3,112 @@
 title: Add user, edit user, your account
 ---
 
-<div class="mw6 center mv6">
-  <h1>This will show states for:</h1>
-  <ul>
-    <li>Add user (only available to Admin user)</li>
-    <li>Edit user (only available to Admin user)</li>
-  </ul>
-</div>
+<main class="mw7 center ph2">
+  <section class="cf mt4 mb3 pa3 pa4-l bg-white shadow br3">
+    <h1 class="mb4 f4 fw6">Add User</h1>
+    <form>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="name" class="db mb2 fw6">
+            Name *
+          </label>
+          <input id="name" type="text" class="w-100 pa3 br2 b--black-10" />
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="email" class="db mb2 fw6">
+            Email *
+          </label>
+          <input id="email" type="text" class="w-100 pa3 br2 b--black-10" />
```

Comment:
Update input type to email: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email

Reply:
Type updated to "email"